.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
+        environment:
+          FPF_CI: true
+          CI_SD_ENV: staging
+          CI_AWS_TYPE: t2.small
+          FPF_GRSEC: false
+          TEST_REPORTS: /root/sd
+    steps:
+      # This is really dumb that Im doing this, basically
+      # I want to clone to ~/pressfreedom and work out of there
+      # the working_directory creates that directory first
+      - run: rm -r ~/sd
+      - checkout:
+          path: ~/sd
+      - run:
+          name: Installation pre-reqs
+          command: |
+            easy_install -U pip
+            pip install pip-tools
+            pip-sync testinfra/requirements.txt
```

Comment:
Since this is a fresh filesystem why not skip installing `pip-tools` and just `pip install -U -r testinfra/requirements.txt`?